IdentityCore/src/oauth2/MSIDTokenResponse.h

```diff
@@ -44,7 +44,10 @@
  */
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->